pyproject.toml

```diff
@@ -60,7 +60,8 @@ dev = [
   "bump-my-version >=0.17.1",
   "codespell",
   "coverage[toml]",
-  "flake8",
+  "coveralls >=4.0.0",
+  "flake8 >=7.0.0",
```

Comment:
I'd add the exact pin on flake8 since you have that in the tox.ini, just so they are the same.